system-addon/content-src/components/Search/_Search.scss

```diff
@@ -184,7 +184,6 @@
   input {
     @include search-input;
     border-top-left-radius: $search-border-radius;
-    border-bottom-left-radius: $search-border-radius;
```

Comment:
I changed this to use the 4-in-1 `border-radius` to avoid issues with accidental ltr partial border styles affecting rtl partial border styles. In this case it would have been okay as the ltr "expanded" bottom-left being 0 happens to match the rtl non-"expanded" bottom-left already being 0.